src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -125,83 +147,123 @@ function ProviderSelectionField({
           <h2 className="vads-u-font-size--h3 vads-u-margin-top--0">
             Choose a provider
           </h2>
-          <p>Your address on file:</p>
-          <ResidentialAddress address={address} />
+          {!loadingProviders && (
+            <>
+              <p>Your address on file:</p>
+              <ResidentialAddress address={address} />
+            </>
+          )}
           {loadingProviders && (
             <div className="vads-u-padding-bottom--2">
-              <LoadingIndicator message="Loading the list of providers" />
+              <LoadingIndicator message="Finding your location. Be sure to allow your browser to find your current location." />
```

Comment:
I think we'd want two different loading messages. One for when we're getting your current location and another for loading providers.

Reply:
Done